tests/integration/__init__.py

```diff
@@ -546,7 +546,10 @@ def list_registered(self, expected_snaps):
         process.close()
         return process.exitstatus
 
-    def get_unique_name(self, prefix='snapcrafttest'):
+    # Do not change this prefix. If you use a different one, it will pollute
+    # KPIs, so always notify the store team if you need to use a different
+    # value.
+    def get_unique_name(self, prefix='test-snapcraft'):
```

Comment:
Since changin the prefix will break KPIs, it seems safer to not make this  configurable at all. I recommend removing this as a parameter from this method.